plugins/module_utils/pulp.py

```diff
@@ -1082,3 +1082,85 @@ def _href(self):
             if self.module.pulp_api.openapi_version == 2
             else "rpm_rpm_repository_href"
         )
+
+
+# Container entities
+
+
+class PulpContainerDistribution(PulpEntity):
+    _list_id = "distributions_container_container_list"
+    _read_id = "distributions_container_container_read"
+    _create_id = "distributions_container_container_create"
+    _update_id = "distributions_container_container_update"
+    _partial_update_id = "distributions_container_container_partial_update"
+    _delete_id = "distributions_container_container_delete"
+
+    _name_singular = "distribution"
+    _name_plural = "distributions"
+
+    @property
+    def _href(self):
```

Comment:
Hmm, maybe we should stop maintaining this terrible backward compatibility hack. But that is a completely different topic. ;)